bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/ui/preferences/LibertyToolsPreferencePage.java

```diff
@@ -0,0 +1,33 @@
+package io.openliberty.tools.eclipse.ui.preferences;
+
+import org.eclipse.core.runtime.preferences.InstanceScope;
+import org.eclipse.jface.preference.BooleanFieldEditor;
+import org.eclipse.jface.preference.DirectoryFieldEditor;
+import org.eclipse.jface.preference.FieldEditorPreferencePage;
+import org.eclipse.jface.preference.RadioGroupFieldEditor;
+import org.eclipse.jface.preference.StringFieldEditor;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchPreferencePage;
+import org.eclipse.ui.preferences.ScopedPreferenceStore;
+
+import io.openliberty.tools.eclipse.LibertyDevPlugin;
+
+public class LibertyToolsPreferencePage extends FieldEditorPreferencePage implements IWorkbenchPreferencePage {
+
+    public LibertyToolsPreferencePage() {
+        super(GRID);
+    }
+
+    public void createFieldEditors() {
```

Comment:
Any thoughts on the "off-by-one-subdir" problem?  E.g. which of the 3 should I specify: ..../mvn, ...../mvn/bin, .../mvn/bin/mvn ?   I think just the second.   This could perhaps be a follow-up validation issue.   I'm not running this as I review, though, just looking at the diff... perhaps you already addressed this and I'm not seeing it.